src/Logging/V2/ConfigServiceV2Client.php

```diff
@@ -56,9 +58,19 @@
  * try {
  *     $configServiceV2Client = new ConfigServiceV2Client();
  *     $formattedParent = ConfigServiceV2Client::formatProjectName("[PROJECT]");
- *     foreach ($configServiceV2Client->listSinks($formattedParent) as $element) {
+ *     // Iterate through all elements
+ *     $pagedResponse = $configServiceV2Client->listSinks($formattedParent);
+ *     foreach ($pagedResponse->iterateAllElements() as $element) {
  *         // doThingsWith(element);
  *     }
+ *
+ *     // OR iterate over pages of elements, with the maximum page size set to 5
+ *     $pagedResponse = $configServiceV2Client->listSinks($formattedParent, ['pageSize' => 5]);
+ *     foreach ($pagedResponse->iteratePages() as $page) {
+ *         foreach ($page as $element) {
+ *             // doThingsWith(element);
```

Comment:
Apologies for having missed this before, but small nit here: could this be `doThingsWith($element)`?